lib/context.ml

```diff
@@ -1,10 +1,17 @@
 open Base
 open Ast_intf
 
-module TypeVar = String
+module Key = struct
+  module T = struct
+    type t = Var of String.t | MFA of String.t * String.t * Int.t
```

Comment:
To find types from PLT, I extended `TypeVar` and renamed `TypeVar` to `Key` because no longer it is only type variable.